app.py

```diff
@@ -24,24 +24,34 @@
 
 # Handle Cloud Foundry with Sentinel
 if 'VCAP_SERVICES' in os.environ:
-  services = json.loads(os.getenv('VCAP_SERVICES'))
-  service = services.get('redislabs')[0]
-  creds = service['credentials']
-  redis_password = creds['password']
-  if not os.getenv('NO_URL_QUOTING'):
-      redis_password = quote(redis_password, safe='')
-  app.config['REDIS_URL'] = 'redis+sentinel://:%s@%s:%s/%s/0' % (
-    redis_password,
-    creds['sentinel_addrs'][0],
-    creds['sentinel_port'],
-    quote(creds['name'], safe=''))
+    services = json.loads(os.getenv('VCAP_SERVICES'))
+    service = services.get('redislabs')[0]
+    creds = service['credentials']
+    redis_password = creds['password']
+    if not os.getenv('NO_URL_QUOTING'):
+        redis_password = quote(redis_password, safe='')
+
+    if 'sentinel_addrs' in creds:
+        sentinel_addr = creds['sentinel_addrs'][0]
+        sentinel_port = creds['sentinel_port']
+    else:
+        sentinel_addr = os.getenv('REDIS_SENTINEL_HOST')  # example: 1.1.1.1,2.2.2.2
+        sentinel_port = os.getenv('REDIS_SENTINEL_PORT')
+
+    app.config['REDIS_URL'] = 'redis+sentinel://:%s@%s:%s/%s/0' % (
+        redis_password,
+        sentinel_addr,
+        sentinel_port,
+        quote(creds['name'], safe=''))
+
 
 if 'REDIS_SENTINEL_HOST' in os.environ:
```

Comment:
@gil-rl It might be better to have an `elif` clause here instead of just if.
The 2nd if looks for `REDIS_SENTINEL_HOST` and will override your `REDIS_URL` config.